modules/gitops-release-notes-1-12-3.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-3_{context}"]
+= Release notes for {gitops-title} 1.12.3
+
+{gitops-title} 1.12.3 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.3_{context}"]
+== Errata updates
+
+[id="rhsa-2024:3368-04-gitops-1-12-3-security-update-advisory_{context}"]
+=== RHSA-2024:3368-04 - {gitops-title} 1.12.3 security update advisory
+
+Issued: 2024-05-28
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3368[RHSA-2024:3368-04]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
```

Comment:
What is this change about ?

Reply:
This is the normal text that's used for all the errata releases @iam-veeramalla. Adding a reference here:

https://docs.openshift.com/gitops/1.10/release_notes/gitops-release-notes.html#errata-updates-1-10-4_gitops-release-notes

Reply:
Fixed.